docs/Users_Guide/skew_t.rst

```diff
@@ -0,0 +1,157 @@
+   environment variables.
+
+   The configuration file is configured to plot the pressure levels adjacent to the temperature line:
+   *level_labels_along_y2-axis: False*
+
+   To plot the pressure levels along the y2-axis (right side of the plot), set this value to *True*:
+   *level_labels_along_y2-axis: True*
+
+   The density of wind barbs has been reduced to plot every second wind barb by setting
+   **decimate_barbs: 2**.  If no decimation/thinning of wind barbs is required, set this value to 1.
+
+
+   To generate the skew-T plots for all available sounding hours, set *all_sounding_hours:True*
+
+   To generate plots for specific sounding hours, set *all_sounding_hours:False* and
+   provide a list of all the hours of interest via the sounding_hours_of_interest setting:  *sounding_hours_of_interest: [6,18,30]*.
+   Replace these hours in the example with all the hours you wish to plot.
+
+
+Running the plotter from the Command Line
+=========================================
+
+To generate a skew-T plot from the sample data and your
+**skew_t.yaml** configuration file, perform the following steps:
+
+* If using the conda environment,
+  verify the conda environment is running and has the required
+  Python packages outlined in the `requirements section
+  <https://metplotpy.readthedocs.io/en/latest/Users_Guide/installation.html#python-requirements>`_.
+
+* Set the METPLOTPY_BASE environment variable to point to
+  *$METPLOTPY_BASE*
+
+  For the ksh environment:
+
+  .. code-block:: ini
+
+    export METPLOTPY_BASE=$METPLOTPY_BASE
+
+  For the csh environment:
+
+  .. code-block:: ini
+
+    setenv METPLOTPY_BASE $METPLOTPY_BASE
+
+  Recall that $METPLOTPY_BASE is the directory path indicating where the METplotpy
+  source code was saved.
+
+
+* Enter the following command:
+
+  .. code-block:: ini
+
+    python $METPLOTPY_BASE/metplotpy/plots/skew_t/skew_t.py $WORKING_DIR/skew_t.yaml
+
+*  Numerous skew-T output files will be created in the directory that was
+   specified in the *output_directory* configuration setting in
+   **skew_t.yaml**. NOTE: The sample data contain some missing
+   values for certain sample hours. Plots are only generated for those hours where there is data.
+   A warning message is logged to the log file indicating which hour has missing data.
+   When data are comprised entirely of missing data, then no corresponding plot will be generated.
+   A warning message is logged to the log file indicating which file contains no data.
+
+   Output files are named with the same filename as the input file, with the *.png* extension replacing the
+   *.dat* extension of the data file. The
+   plot title is generated from the name of the input file and the corresponding sounding hour.
+
+Example output plot
+___________________
+
+The **skew_t.yaml** configuration file in combination with the
+Python script and sample data, generates the following skew-T plot:
+.. image:: figure/ssh052023_avno_doper_2023010100_diag_0_hr.png
```

Comment:
The image does not seem to render in RTD: 

![image](https://github.com/dtcenter/METplotpy/assets/19677014/cdab2ec8-b7a1-4702-93ad-31e710b392bd)

Is there a syntax issue with how it is being included?

Reply:
needed to add another carriage return in between the previous line and the image syntax.  Now the image is rendering.